stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -32,6 +45,10 @@
 
   private final Context mContext;
 
+  private Map<String, AsyncPrettyPrinter> mRequestIdMap = Collections.synchronizedMap(
+      new HashMap<String, AsyncPrettyPrinter>());
+  private ExecutorService sExecutor = Executors.newSingleThreadExecutor();
```

Comment:
Prefer unifying all of your executors into one and documenting it in a holder class that stands on its own.  Also, I highly recommend you use parallel execution (not a single threaded executor) for better performance.